utilityscripts/tiffs/tiffconvert.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
```

Comment:
Suggest running the file through PEP8 (https://www.python.org/dev/peps/pep-0008/)